drivers/hmis/app/graphql/types/forms/form_identifier.rb

```diff
@@ -0,0 +1,45 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Types
+  class Forms::FormIdentifier < Types::BaseObject
+    skip_activity_log
+    description 'FormIdentifier'
+
+    available_filter_options do
+      arg :search_term, String
+      # ADD: role
+      # ADD: status
+    end
+
+    # TODO: add summary information so the UI can resolve top-level detail such as 'active in 15 projects'
```

Comment:
@gigxz should this update, and other updates to basically make the form versions table match the new [design](https://www.figma.com/file/F9EHR2IZYuMAlOJ9aOG2IL/OP-%2F-HMIS-%2F-2024-%2F-Q1-%2B-Q2-(Jan-1---June-30)?type=design&node-id=2611-31087&mode=design&t=s8EuJodzWywXzhGa-0), be in-scope for this ticket?

Reply:
the "summary" piece needs to be spec'd still, so no. We need to think more about what is helpful to show imo. If there are UI-only changes on the forms table, feel free to make those.